src/content/translations/bn/desci/index.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "hello\nthis is afnan"
```

Comment:
Not sure who afnan is, but I don't think this is correct.

```suggestion
title: বিকেন্দ্রীভূত বিজ্ঞান
```

(Taken from content below, confirmed with Google)

<img width="499" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/d2864209-3813-4349-a69f-ad64172256de">